html-web-page/index.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>HTML Web Page</title>
+    <style>
+      table,
+      th,
+      td {
+        border: 1px solid black;
+      }
+
+      td:not(:first-child) {
+        text-align: center;
+      }
+    </style>
+  </head>
+  <body>
+    <h1>Jelly of the Month Club</h1>
+    <h2>Make all of your friends jelly of your stock of jelly!</h2>
+
+    <p>
+      Here at JOTMC Inc., we are constantly striving to supply our customers
+      with the highest-quality and most unique jellies on a monthly basis. For
+      the jellly lover in your life (or yourself!), this is the gift that keeps
+      on giving all year round. We offer three tiers of monthly jellly joy, and
+      each one offers their own perks. Here is a breakdown of each, as well as
+      your monthly price:
+    </p>
+    <!-- your HTML code here 👇 -->
+
+    <table>
+      <thead>
+        <th></th>
+        <th>Basic Plan</th>
+        <th>Deluxe Plan</th>
+        <th>Jelly Giant Plan</th>
+      </thead>
+      <tbody>
+        <tr>
+          <td>Three new curated jellies per month</td>
+          <td>&check;</td>
+          <td>&check;</td>
+          <td>&check;</td>
+        </tr>
+        <tr>
+          <td>Monthly jellies come with commemorative mason jars</td>
+          <td></td>
+          <td>&check;</td>
+          <td>&check;</td>
+        </tr>
+        <tr>
+          <td>Free bonus jar of preserves bi-monthly</td>
+          <td></td>
+          <td>&check;</td>
+          <td>&check;</td>
+        </tr>
+        <tr>
+          <td>Free yearly tickets to Jelly Con</td>
+          <td></td>
+          <td></td>
+          <td>&check;</td>
+        </tr>
+        <tr>
+          <td>Monthly Pricing</td>
+          <td>$30</td>
+          <td>$50</td>
+          <td>$60</td>
+        </tr>
+      </tbody>
+    </table>
+
+    <p>Like what you see? Become a part of the JOTMC family today!</p>
+    <p>
+      <input
+        type="text"
+        id="your-name"
+        name="your-name"
+        placeholder="Your Name" />
+      <!-- <textarea id = "your-name" name="your-name" rows="1">Your Name</textarea> -->
+      <select>
+        <option selected value="Select Your Plan:">Select Your Plan:</option>
+        <option value="basic">Basic Plan</option>
+        <option value="deluxe">Deluxe Plan</option>
+        <option value="premium">Jelly Giant Plan</option>
+      </select>
+      <button type="submit">Submit</button>
+    </p>
```

Comment:
Instead of using a `p`, you should use a `div`. `p` elements are only for when we want to have text inside. If you need an element to "contain" the rest of the content, that's where a `div` would come into play